app/controllers/comparisons/base_controller.rb

```diff
@@ -43,6 +45,31 @@ def create_charts(_results)
       []
     end
 
+    def create_single_number_chart(results, name, series_name, y_axis_label)
+      chart_data = {}
+
+      # Some charts also set x_max_value to 100 if there are metric values > 100
+      # Removes issues with schools with large % changes breaking the charts
+      #
+      # This could be done by clipping values to 100.0 if the metric has a
+      # unit of percentage/relative_percent
+      results.each do |result|
+        metric = result.send(name)
+        next if metric.nil? || metric.nan? || metric.infinite?
+
+        # for a percentage metric we'd multiply * 100.0
+        # here we're converting from kW to W
+        chart_data[result.school.name] = metric * 1000.0
```

Comment:
This will need revising as its only baseload and maybe peak usage where we convert to watts. As noted in the comments in other places we convert percentages.